internal/lib/lib.go

```diff
@@ -1,112 +1,19 @@
-	"io"
-	"os"
-	"strings"
 
-	log "github.com/sirupsen/logrus"
-
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/artifacts"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/engine"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/formatters"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/policy"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
 )
 
-// CheckContainerRunner contains all of the components necessary to run checkContainer.
-type CheckContainerRunner struct {
-	Cfg       *runtime.Config
-	Pc        PyxisClient
-	Eng       engine.CheckEngine
-	Formatter formatters.ResponseFormatter
-	Rw        ResultWriter
-	Rs        ResultSubmitter
-}
-
-func NewCheckContainerRunner(ctx context.Context, cfg *runtime.Config, submit bool) (*CheckContainerRunner, error) {
-	cfg.Policy = policy.PolicyContainer
-	cfg.Submit = submit
-
-	pyxisClient := NewPyxisClient(ctx, cfg.ReadOnly())
-	// If we have a pyxisClient, we can query for container policy exceptions.
-	if pyxisClient != nil {
-		policy, err := GetContainerPolicyExceptions(ctx, pyxisClient)
-		if err != nil {
-			return nil, err
-		}
-
-		cfg.Policy = policy
-	}
-
-	engine, err := engine.NewForConfig(ctx, cfg.ReadOnly())
-	if err != nil {
-		return nil, err
-	}
-
-	fmttr, err := formatters.NewForConfig(cfg.ReadOnly())
-	if err != nil {
-		return nil, err
-	}
-
-	rs := ResolveSubmitter(pyxisClient, cfg.ReadOnly())
-
-	return &CheckContainerRunner{
-		Cfg:       cfg,
-		Pc:        pyxisClient,
-		Eng:       engine,
-		Formatter: fmttr,
-		Rw:        &runtime.ResultWriterFile{},
-		Rs:        rs,
-	}, nil
-}
-
-// CheckOperatorRunner contains all of the components necessary to run checkOperator.
-type CheckOperatorRunner struct {
-	Cfg       *runtime.Config
-	Eng       engine.CheckEngine
-	Formatter formatters.ResponseFormatter
-	Rw        ResultWriter
-}
-
-// NewCheckOperatorRunner returns a CheckOperatorRunner containing all of the tooling necessary
-// to run checkOperator.
-func NewCheckOperatorRunner(ctx context.Context, cfg *runtime.Config) (*CheckOperatorRunner, error) {
-	cfg.Policy = policy.PolicyOperator
-	cfg.Submit = false // there's no such thing as submitting for operators today.
-
-	engine, err := engine.NewForConfig(ctx, cfg.ReadOnly())
-	if err != nil {
-		return nil, err
-	}
-
-	fmttr, err := formatters.NewForConfig(cfg.ReadOnly())
-	if err != nil {
-		return nil, err
-	}
-
-	return &CheckOperatorRunner{
-		Cfg:       cfg,
-		Eng:       engine,
-		Formatter: fmttr,
-		Rw:        &runtime.ResultWriterFile{},
-	}, nil
-}
-
-// ResolveSubmitter will build out a ResultSubmitter if the provided pyxisClient, pc, is not nil.
-// The pyxisClient is a required component of the submitter. If pc is nil, then a noop submitter
-// is returned instead, which does nothing.
```

Comment:
Should we keep this comment?

Reply:
Yep, sorry - I moved the old functions into different places so I could use the same function names and migrate to them. Some comments got caught in the crossfire.